### PR TITLE
Remove SVN from Mx10 docs

### PR DESCRIPTION
**Pull request by ConnorLand**
Copied over from this private PR: https://github.com/mendix/Mendix10-docs/pull/8

Please replicate all comments yourself here, as they cannot be ported over.

### CODE REVIEW

File: content/en/docs/refguide/modeling/menus/version-control-menu/download-from-version-control-dialog.md [hunk 1]
```diff
@@ -23,17 +23,17 @@ You can also use the [Open App Dialog](/refguide/open-app-dialog/) to download a
 
 ## 2 Where Is Your App Stored?
 
-If **Enable private version control** (either for SVN or Git or for both) is set in the app [Preferences](/refguide/preferences-dialog/#enable), you can choose between the **Mendix Team Server** or a **Private server**. If it is not enabled, you will only be able to choose an app from the Mendix Team Server.
+If **Enable private version control with Git** is set in the app [Preferences](/refguide/preferences-dialog/#enable), you can choose between the **Mendix Team Server** or a **Private server**. If it is not enabled, you will only be able to choose an app from the Mendix Team Server.
 
 ### 2.1 Mendix Team Server
 
-Use the **Team Server App** dropdown to choose the app you want to download.
+Use the **Team Server App** dropdown to choose the Git app you want to download.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/menus/version-control-menu/download-from-version-control-dialog.md:30]: @maksymvyhovskyi , I added "Git" to this line for clarity.

File: content/en/docs/refguide/modeling/menus/version-control-menu/branch-line-manager-dialog/_index.md [hunk 2]
```diff
@@ -20,19 +20,19 @@ A branch line allows independent development from other development lines. There
...
 
 {{% alert color="warning" %}}
-This option is only available when support for private SVN or Git servers is enabled in [Preferences](/refguide/preferences-dialog/).
+This option is only available when support for private Git servers is enabled in [Preferences](/refguide/preferences-dialog/).
 {{% /alert %}}
 
 ### 2.1 Team Server App {#team-server-app}
 
-Select the Team Server app for which you want to manage the branch lines. If you have an app open in Studio Pro, it will be selected automatically. However, you can also manage branch lines without opening an app first, in which case no app will be selected.
+Select the Team Server Git app for which you want to manage the branch lines. If you have an app open in Studio Pro, it will be selected automatically. However, you can also manage branch lines without opening an app first, in which case no app will be selected.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/modeling/menus/version-control-menu/branch-line-manager-dialog/_index.md:31]: @maksymvyhovskyi , I added "Git" to this line for clarity.

File: content/en/docs/refguide/modeling/menus/version-control-menu/download-from-version-control-dialog.md (hunk 1 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/modeling/menus/version-control-menu/download-from-version-control-dialog.md:30]: Verified via Slack.

File: content/en/docs/refguide/modeling/menus/version-control-menu/branch-line-manager-dialog/_index.md (hunk 2 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/modeling/menus/version-control-menu/branch-line-manager-dialog/_index.md:31]: Verified via Slack.

**Review by MariaShaposhnikova — APPROVED** [review]

**Comment by MarkvanMents** [comment]
@ConnorLand 
I see that some of the changes in this PR are to unversioned documents in the AppStore.
I think some of the changes which remove Git entirely might be too much as many customers will not be migrating to Git for some time and SVN is still available in previous versions of Mendix.

Can you have a look at this again - I can discuss it with you if necessary.

**Comment by MarkvanMents** [comment]
> @ConnorLand I see that some of the changes in this PR are to unversioned documents in the AppStore. I think some of the changes which remove Git entirely might be too much as many customers will not be migrating to Git for some time and SVN is still available in previous versions of Mendix.
> 
> Can you have a look at this again - I can discuss it with you if necessary.

Actually - have looked again and it seems to be OK.

**Pull request merged**